k8s/upgrade/src/upgrade_resources/upgrade.rs

```diff
@@ -667,31 +673,33 @@ pub async fn get_pvc_from_uuid(uuid_list: HashSet<String>) -> Result<Vec<String>
 }
 
 /// Return results as list of deployments.
-pub async fn get_deployment_for_rest(ns: &str) -> Result<ObjectList<Deployment>, Error> {
+pub async fn get_deployment_for_rest(ns: &str) -> Result<Deployment, Error> {
     let client = Client::try_default().await?;
     let deployment = Api::<Deployment>::namespaced(client.clone(), ns);
```

Comment:
client is not used again, clone not needed?

Reply:
Removed some of the extra clones..